documentation/src/main/asciidoc/repositories/Repositories.adoc

```diff
@@ -563,4 +563,4 @@ List<Book> books(@By("isbn") String[] ibsns);
 
 Naturally, the name and type of the parameter are still checked at compile time; there's no loss of typesafety here, despite the string.
 
-The `@Param` annotation is significantly less useful, since we can always rename our HQL query parameter to match the method parameter, or, at worst, use an ordinal parameter instead.
\ No newline at end of file
+The `@Param` annotation is significantly less useful, since we can always rename our HQL query parameter to match the method parameter, or, at worst, use an ordinal parameter instead.
```

Comment:
@cunhazera What changed here ... the lines look identical.

Reply:
@gavinking no idea... I don't see any differences in any diff checker except GitHub's. But I can send a new commit trying to undo this change.